Teacher Workout/Features/Home/ViewController.swift

```diff
@@ -1,3 +1,12 @@
 import UIKit
 
-class ViewController: UIViewController {}
+class ViewController: UIViewController {
+    #warning("remove this")
```

Comment:
On the main branch, we now have the AppCoordinator. We should trigger the onboarding from there.

Reply:
Also, please add the logic to display the onboarding only once.